src/utils/SVGIcons.ts

```diff
@@ -10,6 +10,8 @@ export class SVGIcons {
     arrowDown: require(`svg/arrow-down`),
 
     mainClear: require(`svg/main-clear`),
+    clearSmall: require(`svg/clear-small`),
```

Comment:
This icon is part of [Vapor's icons](https://vapor.cloud.coveo.com/#/styles/icons/list).